Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -938,7 +940,12 @@ extension HTTPClient.Configuration {
         /// - parameters:
         ///     - connect: `connect` timeout. Will default to 10 seconds, if no value is provided.
         ///     - read: `read` timeout.
-        public init(connect: TimeAmount? = nil, read: TimeAmount? = nil) {
+        ///     - write: `write` timeout.
+        public init(
+            connect: TimeAmount? = nil,
+            read: TimeAmount? = nil,
+            write: TimeAmount? = nil
+        ) {
```

Comment:
This is a breaking change:
```swift
15:21:52 1 breaking change detected in AsyncHTTPClient:
15:21:52   💔 API breakage: constructor HTTPClient.Configuration.Timeout.init(connect:read:) has been removed
```
Adding a new parameter, even with a default, is breaking because users can write:
```
// type is inferred as `(Timeout?, Timeout?) -> Timeout`
let createTimeout = HTTPClient.Configuration.Timeout.init
let timeout = createTimeout(nil, nil)
```
with you change the inferred type is now `(Timeout?, Timeout?, Timeout?) -> Timeout` which is breaking.

The workaround is to leave the old init as is and default to `nil` for `write`. Add a new init where the `write` parameter is required.